environment.yml

```diff
@@ -32,5 +32,7 @@ dependencies:
   - rpyc==4.1.5
   - cloudpickle==1.4.1
   - boto3
+  - curl
   - pip:
     - ray>=1.0.0
+    - PyGithub
```

Comment:
`curl` and `PyGithub` are package specific for internal testing, let's install them explicitly in dockerfile.

Reply:
Ok I moved `curl` and `PyGithub` to docker file.